nimare/base/estimators.py

```diff
@@ -2,12 +2,9 @@
 Base classes for datasets.
 """
 import logging
-from abc import ABCMeta, abstractmethod
```

Comment:
Isn't `ABCMeta` necessary? Haven't double-checked this, but I think `abstractmethod` will only work on classes that have `ABCMeta` as their metaclass.

Reply:
I think that both Estimator and Transformer inherit from NiMAREBase, which has ABCMeta as its metaclass.